search-services/1.3/using/index.md

```diff
@@ -289,13 +289,14 @@ Single terms, phrases, and so on can be combined using "`NOT`" in upper, lower,
 These queries search for nodes that contain the terms `yellow` in any content.
 
 ```sql
-yellow NOT banana
-yellow !banana
-yellow -banana
+yellow AND NOT banana
+yellow AND !banana
+yellow AND -banana
 NOT yellow banana
```

Comment:
Add line here "These queries search for nodes that contain the term `banana` but do not contain "yellow" in any content."